src/internal/cmds.ts

```diff
@@ -36,15 +37,15 @@ export class CmdUtils {
   static asSelection(data: {idtype: string, selection: string}): ISelection {
     return {
       range: data.selection ? ParseRangeUtils.parseRangeLike(data.selection) : Range.none(),
-      idtype: data.idtype ? IDTypeManager.getInstance().resolveIdType(data.idtype) : null
+      idtype: data.idtype ? IDTypeManager.getInstance().resolveIdType(data.idtype) : new IDType('Start', 'Start', '', true)
```

Comment:
`Start` is not a correct idtype and also results in a wrong frame around the view. 

**Wrong:**

![grafik](https://user-images.githubusercontent.com/5851088/135917408-fb22fd84-301e-4ef0-823c-85bb52f1fbf3.png)

**Correct:** 

![grafik](https://user-images.githubusercontent.com/5851088/135917538-d7298c2f-9553-4ae6-8cd9-64c1fec86e48.png)

We should fix the idtype in the start ranking views and pass the correct idtypes to the `data` property and not try to fix it here.